WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -103,6 +103,10 @@ def index
       end
     end
 
+    if (1..4).cover? params[:days].to_i
```

Comment:
Maybe we'll have competitions spanning one week in the future (I hope not :P), so this would need to change.